web/styles/recent_topics.css

```diff
@@ -74,16 +74,29 @@
 
         .table_fix_head {
             padding: 0 !important;
-            /* 100px = space occupied by `recent_topics_filter_buttons`( ~49px)
-                     + give user some extra space at the bottom so that last
-                       topic row is clearly visible. */
-            max-height: calc(100vh - 100px);
+            /*
+            42px = approx height of closed compose box.
+            12px = top padding of #recent_topics_table.
+            */
+            max-height: calc(
+                100vh - var(--recent-topics-filters-height) -
+                    var(--navbar-fixed-height) - 42px - 12px
+            );
```

Comment:
Is it possible for this calculation to be negative on small windows?

Reply:
Doesn't happen on iphone SE which has super small height.
<img width="340" alt="image" src="https://github.com/zulip/zulip/assets/25124304/6110c9a6-6bce-47e7-828d-6440b356aee9">

Reply:
https://chat.zulip.org/#narrow/stream/101-design/topic/Small.20device.20icons for compose icons.

Reply:
Well the case I'd imagine is a phone in landscape mode, but I'm not sure even that would fail in this way, and it's not a mode we test currently anyway.